packages/frontend/src/components/SimpleStatistic/BigNumberContextMenu.tsx

```diff
@@ -39,13 +42,13 @@ export const BigNumberContextMenu: FC<BigNumberContextMenuProps> = ({
         [bigNumberConfig],
     );
 
-    const row: ResultRow = useMemo(() => {
-        return resultsData?.rows?.[0] || {};
+    const row: UnderlyingValueMap = useMemo(() => {
+        return resultsData?.rows?.[0].value?.value || {};
```

Comment:
Looks like there is a bug with big numbers. In this video I show the options working well in another PR env, and not working in this PR env.

https://user-images.githubusercontent.com/9117144/233040945-a77ae856-cf78-41ba-9c6c-22b4f2806c04.mp4




Should this be similar to the code in other components? 
```const row = mapValues(cell.row.original, (v) => v?.value) || {};```